src/Service.Tests/Snapshots/ConfigurationTests.TestReadingRuntimeConfigForCosmos.verified.txt

```diff
@@ -226,7 +226,11 @@
               }
             ]
           }
-        ]
+        ],
+        Cache: {
+          Enabled: false,
```

Comment:
None of these changes to the Snapshots are required. Since we should not create a new object if the user didnt specify explicit values to the cache options. 

CacheOptions object should be created only when they are specified by the user